src/components/Search/FilterDropdowns/UserSelectPopup.tsx

```diff
@@ -124,7 +124,17 @@ function UserSelectPopup({value, closeOverlay, onChange}: UserSelectPopupProps)
             };
         });
 
-        return [...personalDetailList, ...recentReportsList];
+        const currentUserOption = filteredOptions.currentUserOption;
+        const userOptions = currentUserOption
+            ? [
+                  {
+                      ...currentUserOption,
+                      isSelected: selectedOptions.some((selectedOption) => selectedOption.accountID === currentUserOption?.accountID),
+                  },
+              ]
+            : [];
+
+        return [...userOptions, ...personalDetailList, ...recentReportsList];
```

Comment:
```suggestion
        const currentUserOption = filteredOptions.currentUserOption
            ? [
                  {
                      ...filteredOptions.currentUserOption,
                      isSelected: selectedOptions.some((selectedOption) => selectedOption.accountID === filteredOptions.currentUserOption?.accountID),
                  },
              ]
            : [];

        return [...currentUserOption, ...personalDetailList, ...recentReportsList];
```

NAB. Just to avoid having two constants with similar names